### PR TITLE
Added files for using docker and updated readme.

### PR DESCRIPTION
**Pull request by bobonthenet**

### CODE REVIEW

File: .docker/web/run.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
Comment (James-Firth) [inline .docker/web/run.sh]: May as well exit early if any step fails, right? :) 
```suggestion
#!/bin/sh -e
```

**Comment by bobonthenet** [comment]
Thanks for the suggestion. I'm a total noob with docker but I need to learn it for future projects at work. Any input is welcome.

**Comment by James-Firth** [comment]
No problem, I use a lot of docker and bash stuff at work 😄 
You could also replace `step` with `echo` (auto-appends a `\n`) if you wanted.

I'm not familiar enough with apache or php to really comment on those parts, but they look fine to me!

**Comment by pspeter3** [comment]
Was the intention for this container to be used in production or development? Also, I think some configuration/documentation is missing because it defaults to 404.

**Pull request merged**